addons/website/views/res_config_settings_views.xml

```diff
@@ -20,23 +20,33 @@
         <field name="arch" type="xml">
             <xpath expr="//form" position="inside">
                 <app data-string="Website" string="Website" name="website" groups="website.group_website_designer">
-                    <setting type="header" string="Settings of Website">
+                    <setting
+                        type="header"
+                        string="Settings of Website"
+                        documentation="/applications/websites/website/configuration/multi_website.html"
+                    >
                         <field name="website_id" options="{'no_open': True, 'no_create': True}" title="Settings on this page will apply to this website"/>
                         <button name="action_website_create_new" type="object" string="+ New Website" class="btn btn-link"/>
                     </setting>
                     <block title="Website Info" id="website_info_settings">
                         <setting>
                             <div class="content-group">
                                 <div class="row mt8">
-                                    <label class="col-lg-3" string="Domain" for="website_domain"/>
+                                    <div class="col-lg-3">
```

Comment:
we can't do that in stable, don't change DOM structure it will break xpath
honestly, just retarget to master, and probably then need for the framework to correctly handle those old settings, as discussed here https://discord.com/channels/678381219515465750/687338039717920792/1218166440314540082